kevm-pyk/src/tests/integration/test_conformance.py

```diff
@@ -30,14 +30,18 @@
 TEST_DIR: Final = REPO_ROOT / 'tests/ethereum-tests'
 GOLDEN: Final = (REPO_ROOT / 'tests/templates/output-success-llvm.json').read_text().rstrip()
 TEST_FILES_WITH_CID_0: Final = (REPO_ROOT / 'tests/bchain.0.chainId').read_text().splitlines()
+FAILING_TESTS_FILE: Final = REPO_ROOT / 'tests/failing.llvm'
+SLOW_TESTS_FILE: Final = REPO_ROOT / 'tests/slow.llvm'
 
 
-def _test(gst_file: Path, schedule: str, mode: str, usegas: bool) -> None:
+def _test(gst_file: Path, schedule: str, mode: str, usegas: bool, save_failing: bool = False) -> None:
```

Comment:
For better readability:

```suggestion
def _test(gst_file: Path, *, schedule: str, mode: str, usegas: bool, save_failing: bool = False) -> None:
```